src/trix/elements/trix_editor_element.coffee

```diff
@@ -146,6 +146,10 @@ Trix.registerElement "trix-editor", do ->
         @parentNode.insertBefore(element, this)
         element
 
+  form:
+    get: ->
+      @inputElement?.form || findClosestElementFromNode(this, matchingSelector: "form")
```

Comment:
https://github.com/rails/rails/pull/42069 has merged. Are we sure we don't want to also support checking ancestors for a `<form>`?

Reply:
> Are we sure we don't want to also support checking ancestors for a `<form>` when an `<input>` isn't declared or associated?

Yes, I don't think we should suggest that a `<trix-editor>` is form-associated when it really isn't. 

FYI, an `<input>` is created automatically when one isn't declared (via the `"input"` attribute): https://github.com/basecamp/trix/blob/66d26af0fdec6424096459047c964a3f9c16593b/src/trix/elements/trix_editor_element.coffee#L149-L158